source/sslnew.c

```diff
@@ -129,6 +129,7 @@ int main(int argc, char *argv[])
     else
         printf("Average time per SSL/BIO creation call: %lfus\n",
                avcalltime);
+    rc = 0;
```

Comment:
Better would be to use `rc = EXIT_SUCCESS` (since we initialise it with `EXIT_FAILURE`)